.cicdtemplate/.codemagic/codemagic.yaml

```diff
@@ -97,3 +99,67 @@ workflows:
       - *unit_test_on_template_xml
     artifacts:
       - *artifacts_template_xml
+
+  build-and-deploy-template-compose-staging:
+    name: Build and deploy template-compose staging to Firebase App Distribution
+    <<: *configure_environment
+    environment:
+      groups:
+        - firebase_credentials
+        - build_configure
+    triggering:
+      events:
+        - push
+      branch_patterns:
+        - pattern: develop
+        - pattern: main
```

Comment:
I don't think we should distribute staging build on main. How about distributing staging on develop and production on main?

Reply:
@luongvo Sounds great. Let me update it. FYI, we need to update this to the github action later. 🙏🏻